src/v/debug_bundle/debug_bundle_service.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "base/seastarx.h"
+#include "config/property.h"
+#include "debug_bundle/error.h"
+#include "debug_bundle/types.h"
+#include "utils/uuid.h"
+
+#include <seastar/core/sharded.hh>
+
+namespace debug_bundle {
+
+/**
+ * @brief Service used to manage creation of debug bundles
+ *
+ * This service is used to create, delete, and manage debug bundles using the
+ * "rpk debug bundle" application
+ */
+class debug_bundle_service final
```

Comment:
`debug_bundle::debug_bundle_service` feels a bit redundant.

Reply:
fair... service is probably just fine

Reply:
or get rid of the namespace